static/app/views/app/index.tsx

```diff
@@ -270,25 +272,40 @@ function App({children, params}: Props) {
       <LastKnownRouteContextProvider>
         <RouteAnalyticsContextProvider>
           {renderOrganizationContextProvider(
-            <AsyncSDKIntegrationContextProvider>
-              <GlobalFeedbackForm>
-                <GlobalDrawer>
-                  <MainContainer tabIndex={-1} ref={mainContainerRef}>
-                    <GlobalModal onClose={handleModalClose} />
-                    <SystemAlerts className="messages-container" />
-                    <Indicators className="indicators-container" />
-                    <ErrorBoundary>{renderBody()}</ErrorBoundary>
-                  </MainContainer>
-                </GlobalDrawer>
-              </GlobalFeedbackForm>
-            </AsyncSDKIntegrationContextProvider>
+            <ChonkThemeProvider>
+              <AsyncSDKIntegrationContextProvider>
+                <GlobalFeedbackForm>
+                  <GlobalDrawer>
+                    <MainContainer tabIndex={-1} ref={mainContainerRef}>
+                      <GlobalModal onClose={handleModalClose} />
+                      <SystemAlerts className="messages-container" />
+                      <Indicators className="indicators-container" />
+                      <ErrorBoundary>{renderBody()}</ErrorBoundary>
+                    </MainContainer>
+                  </GlobalDrawer>
+                </GlobalFeedbackForm>
+              </AsyncSDKIntegrationContextProvider>
+            </ChonkThemeProvider>
           )}
         </RouteAnalyticsContextProvider>
       </LastKnownRouteContextProvider>
     </Profiler>
   );
 }
 
+function ChonkThemeProvider({children}: {children: React.ReactNode}) {
```

Comment:
Comment for clarity?
```suggestion
/** 
  * Temporary provider for new UI2 theme rollout
  */
function ChonkThemeProvider({children}: {children: React.ReactNode}) {
```